scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -130,6 +130,9 @@ case object RedundantBraces extends Rewrite {
         !isProcedureSyntax(d) &&
         !disqualifiedByUnit
 
+      case p: Term.Function if isBlockFunction(p) =>
```

Comment:
Can we guard this against edition 2019-11? The idea is that people should be able to upgrade to the latest Scalafmt version without having large diffs. Sometimes you want to upgrade to the latest version to enjoy for example a parser fix but need more time to get the formatting changes merged into your codebase.

Reply:
will do